modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js

```diff
@@ -1834,7 +1834,11 @@ AUI.add(
 							layoutValue && layoutValue.privateLayout
 						);
 
-						var groupId = themeDisplay.getScopeGroupIdOrLiveGroupId();
+						var groupId = themeDisplay.getScopeGroupId();
```

Comment:
Can you wrap this logic in a function and just call it where it's needed?

Reply:
I did not want to change the pattern in the file if it was possible and it would not change the behavior of this ticket. So if it's not strictly necessary, I would follow the pattern that was accepted in all previous modifications.

Reply:
Thanks for sharing your concerns. Creating a function would buy us lines and it would be easier to maintain later since all the developer would have to do is calling a function that would return the expected group id, without needing to understand the logic behind the staging state of the portlet, also, if ever this needs to change, the developer would have to fix just the function, this one place, and don't look everywhere in the file to find the same logic.